frontend/ui/lib/widgets/map_widget.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class MapPlaceHolder extends StatelessWidget {
```

Comment:
This could just be called `MapWidget` since the user doesn't need to know that it's just a placeholder.